src/migrations/shoppingCart.js

```diff
@@ -1,7 +1,7 @@
 export const createShoppingCartTable = `
   DROP TABLE IF EXISTS shopping_cart;
   CREATE TABLE IF NOT EXISTS shopping_cart (
-    id INTEGER NOT NULL,
+    id INTEGER UNIQUE NOT NULL,
```

Comment:
Isso é o id do carrinho, ou do produto contido no carrinho? Se for do carrinho, nao deveria ser SERIAL? Se for do produto, nao seria melhor chamar product_id, e colocar uma FOREIGN KEY?

Sintaxe:

```js
create table shopping_cart (
  product_id integer unique not null references product (id)
);
```

O que vem depois do `references` é o nome da tabela, seguido da coluna referenciada, entre parenteses